flags/14.0/vtctld.txt

```diff
@@ -1,84 +1,53 @@
-      --discovery_high_replication_lag_minimum_serving duration          the replication lag that is considered too high when applying the min_number_serving_vttablets threshold (default 2h0m0s)
-      --discovery_low_replication_lag duration                           the replication lag that is considered low enough to be healthy (default 30s)
-      --durability_policy string                                         type of durability to enforce. Default is none. Other values are dictated by registered plugins (default none)
+      --durability_policy string                                         type of durability to enforce. Default is none. Other values are dictated by registered plugins (default "none")
       --emit_stats                                                       If set, emit stats to push-based monitoring and stats backends
-      --enable-autocommit                                                This flag is deprecated. Autocommit is always allowed. (default true)
-      --enable-consolidator                                              Synonym to -enable_consolidator (default true)
-      --enable-consolidator-replicas                                     Synonym to -enable_consolidator_replicas
-      --enable-lag-throttler                                             Synonym to -enable_lag_throttler
-      --enable-query-plan-field-caching                                  Synonym to -enable_query_plan_field_caching (default true)
-      --enable-tx-throttler                                              Synonym to -enable_tx_throttler
-      --enable_consolidator                                              This option enables the query consolidator. (default true)
-      --enable_consolidator_replicas                                     This option enables the query consolidator only on replicas.
-      --enable_hot_row_protection                                        If true, incoming transactions for the same row (range) will be queued and cannot consume all txpool slots.
-      --enable_hot_row_protection_dry_run                                If true, hot row protection is not enforced but logs if transactions would have been queued.
-      --enable_lag_throttler                                             If true, vttablet will run a throttler service, and will implicitly enable heartbeats
-      --enable_queries                                                   [DEPRECATED - query commands via vtctl are being deprecated] if set, allows vtgate and vttablet queries. May have security implications, as the queries will be run from this process.
-      --enable_query_plan_field_caching                                  This option fetches & caches fields (columns) when storing query plans (default true)
       --enable_realtime_stats                                            Required for the Realtime Stats view. If set, vtctld will maintain a streaming RPC to each tablet (in all cells) to gather the realtime health stats.
```

Comment:
we should probably deprecate this, because it is used by the old web UI that is already deprecated.

Reply:
fixed